0.11 compilation error on MacOS clang

- OS version: MacOS 10.15
 - Compiler version: Apple clang version 11.0.3 (clang-1103.0.32.62)
 - PODIO version: 0.11

```
[ 93%] Building CXX object tests/CMakeFiles/read-multiple.dir/read-multiple.cpp.o
/Users/mato/Development/LCGSoft/build/frameworks/podio-00.11/src/podio/00.11/tests/write.cpp:73:23: error: implicit instantiation of undefined template 'std::__1::basic_stringstream<char, std::__1::char_traits<char>, std::__1::allocator<char> >'
    std::stringstream ss ; ss << " event_number_" << i ;
                      ^
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/iosfwd:139:32: note: template is declared here
    class _LIBCPP_TEMPLATE_VIS basic_stringstream;
                               ^
1 error generated.
make[5]: *** [tests/CMakeFiles/write.dir/write.cpp.o] Error 1
make[4]: *** [tests/CMakeFiles/write.dir/all] Error 2
make[4]: *** Waiting for unfinished jobs....
```

--- a/tests/write.cpp
+++ b/tests/write.cpp
@@ -16,6 +16,7 @@
 // STL
 #include <iostream>
 #include <vector>
+#include <sstream>
 
 // podio specific includes
 #include "podio/EventStore.h"

